File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        map<int,int> diffTarget;
        vector<int> returnValue = {-1, -1};
        
        for(int i=0; i<numbers.size(); i++){
            
            map<int,int>::iterator it = diffTarget.find(numbers[i]);
            if(it != diffTarget.end())
            {
                returnValue = {it->second+1, i+1};
                return returnValue;
            }
            
            diffTarget[target-numbers[i]] =  i;
        
        }
        return returnValue;
        
        
    }
};

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        hashmap: Dict = {}
        
        for index, value in enumerate(numbers):
            if target-value not in hashmap:
                hashmap[value] = index
            else:
                return [hashmap[target-value]+1, index+1]
                
        